docs/docs/general/03-Element Selection/scrolling.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 2
+title: Scrolling
+---
+
+# Scrolling
+When you use AskUI you can only interact with elements that you can see on your screen. Therefore you have to scroll down/sideways to interact with currently invisible elements.
+
+### Scrolling With Mouse Wheel
+If you want to scroll with your mouse wheel you can use the `scroll()` action:
+
+```javascript
+// Scroll 10 down in y direction<>
+await aui.scroll(0, -10).exec()
+```
+
+### Scrolling With Key Press
+If you want to scroll with a key press you could use your arrow keys (`up`, `down`, `left`, `right` ) or the `pagedown`-key.
+
+```javascript
+// Press down arrow key
+await aui.pressKey('down').exec()
+
+// Press up arrow key
+await aui.pressKey('up').exec()
+
+// Scroll down a page
+await aui.pressKey('pagedown').exec()
+
+// Scroll up a page
+await aui.pressKey('pageup').exec()
+```
+
+### Scrolling on Touch Displays
+On touch displays you have to recreate the swipe gesture:
+
+```javascript
+// Touch the display with your finger, move finger to the left, release
+await aui.mouseToggleDown().exec()
+await aui.moveMouseRelatively(-1500, 0).exec()
+await aui.mouseToggleUp().exec()
+```
```

Comment:
We have a a `swipe` action. What is going on with that? https://docs.askui.com/docs/api/Actions/swipe

Reply:
Yeah, you would think this works:

`await aui.swipe(500, 0).exec()`

but you can only swipe an element:

`await aui.swipe(500, 0).image().exec()`

Swipe is useful for a different use case. I hope this explains it 😃 .

Reply:
Thx for the explanation :) This clears it up. I would add it here with this explanation as it sound like it may be used if there is a (non-swipeable) element for swiping the whole screen as well. Also we should explain that the mouseToggleDown way here in the example may not work if the mouse is toggled down on a swipeable element.